tests/cpu/UnitTestMain.cpp

```diff
@@ -78,6 +79,7 @@ int main(int argc, const char ** argv)
                "--sse2",          &sse2,             "Enable SSE2 Accelerated features",
                "--avx",           &avx,              "Enable AVX Accelerated features",
                "--avx2",          &avx2,             "Enable AVX2 Accelerated features",
+               "--avx512",        &avx512,           "Enable AVX512 Accelerated features",
                "--f16c",          &f16c,             "Enable F16C Accelerated features",
```

Comment:
Perhaps indicate which of the other options supports the f16c option?

Reply:
Good idea. I can add some clarification on which options support f16c. Only AVX and AVX2 are valid with the f16c. I was thinking of adding f16c emulation like sse2's to AVX. I've been toying around some alternative algorithms trying them make it faster.